src/pages/gateway/command-reference.md

```diff
@@ -21,7 +21,7 @@ aio api-manager:mesh:create [FILE]
 
 ### Arguments
 
-  FILE    The JSON file that contains your mesh's handlers and transforms
+  FILE    The JSON file that contains your mesh's handlers and transforms.
```

Comment:
I don't think the arguments are displaying well, so I'm going to suggest they all be changed.

```suggestion
`FILE` The JSON file that contains your mesh's handlers and transforms.
```